util/get.go

```diff
@@ -0,0 +1,179 @@
+package util
```

Comment:
Don't review this file. It is code that used to be in Terraform's library that they removed but that we still use. It was copied directly from 0.11 so that I wouldn't have to change Terragrunt